com.unity.visualeffectgraph/Editor/Data/VFXDataParticle.cs

```diff
@@ -191,8 +192,188 @@ public BucketInfo[] GetBucketLayoutInfo()
         private List<int> m_BucketOffsets = new List<int>();
     }
 
+    // [NOTE-VFX-MATERIALS]
```

Comment:
Note that this will be removed after the C++ PR lands. It is a mechanism to support faux material serialization to unblock development and allow testing everything else.